extensions/systemd/get-uid.js

```diff
@@ -23,7 +23,9 @@ module.exports = function getUid(dir) {
         return uid;
     } catch (e) {
         if (!e.message.match(/no such user/i)) {
-            throw e;
+            const errors = require('../../lib').errors;
+            // throw the error here so it'll be logged
+            throw new errors.ProcessError(e);
```

Comment:
I thought it might be a better idea to use a `ProcessError` here, as it gets logged at least.

Reply:
Now that I'm thinking about it - is it worth checking the message here at all? could just return null regardless of the error message.

Reply:
> Now that I'm thinking about it - is it worth checking the message here at all? could just return null regardless of the error message. (e.g. as was done in #659)

I agree and also changed the util in #659 to not throw any error. I explained it [here](https://github.com/TryGhost/Ghost-CLI/pull/659#discussion_r174345686) and also in the comment in the codebase [here](https://github.com/TryGhost/Ghost-CLI/pull/659/files#diff-e8c19ddf30935059861a1177570bd9e2R18).

I will refactor the code in the systemd util to do the same.